Dockerfile

```diff
@@ -25,14 +25,38 @@ ENV CONSUL_ADDRESS="" \
     USERS="" \
     EXTRA_FRONTEND=""
 
+# Merge telegraf to this so we can extract stats
+
+ENV TELEGRAF_VERSION 1.1.1
```

Comment:
I haven't used Telegraf (yet) but, from what I've seen, it gathers metrics from HAProxy stats. If that's the case, I think it would be cleaner to run it as a separate container linked through Docker networking. Then again, I might be missing something since I haven't used it yet.